openfl-tutorials/interactive_api/PyTorch_Huggingface_transformers_SUPERB/envoy/sd_requirements.txt

```
@@ -0,0 +1,6 @@
+datasets==1.14
+librosa
+pillow
+pynvml
+torch
+numpy==1.21.5
```

Should we remove useless requirements? (btw, it works for me)
```suggestion
datasets==1.14
librosa
pynvml
numpy==1.21.5
```